cores/arduino/stm32/PortNames.h

```diff
@@ -73,7 +75,9 @@ typedef enum {
 
 #define MAX_NB_PORT (LastPort-FirstPort+1)
 
-GPIO_TypeDef *get_GPIO_Port(uint32_t port_idx);
+/* Return GPIO base address */
+#define get_GPIO_Port(p) ((p < MAX_NB_PORT) ? GPIOPort[p] : (GPIO_TypeDef *)GPIOA_BASE)
```

Comment:
Isn't it better to return `NULL` instead of `GPIOA_BASE` if port is out of bounds?

Reply:
yes. I wonder which value can be the best as if not properly used this would access to 0x00000000. Anyway this could allow to test if it is ok. I will update that.

Reply:
done